### PR TITLE
Auth0 migration blog post

### PR DESCRIPTION
**Pull request by mooreds**
Blog post about auth0 migrations, plus some docs cleanup that came out of writing it.

### CODE REVIEW

File: site/_posts/2021-03-10-migrate-from-auth0.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
+---
+layout: blog-post
+title: What to consider when you are considering an Auth0 migration
```
Comment (dpolstra) [inline site/_posts/2021-03-10-migrate-from-auth0.md:3]: What to think about when you're considering an Auth0 Migration

File: site/_posts/2021-03-10-migrate-from-auth0.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+excerpt_separator: "<!--more-->"
+---
+
+Pssst. You may have heard that [Auth0 was recently acquired by Okta](https://www.okta.com/press-room/press-releases/okta-signs-agreement-to-acquire-auth0/). If this has you considering migration options, read on. This post will provide a strategy for determining if a migration makes sense.
+
+<!--more-->
+
+I'll be using FusionAuth as an example migration alternative, but the investigation and feature mapping can be used to prepare a move to any Auth0 competitor. That said, since the announcement, our website has seen significantly more traffic and we've had dozens of people reach out to learn more about FusionAuth.
+
+## FusionAuth and Auth0
```
Comment (dpolstra) [inline site/_posts/2021-03-10-migrate-from-auth0.md:18]: FusionAuth vs Auth0

File: site/_posts/2021-03-10-migrate-from-auth0.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+* Fully customizable UI
+* A robuse consent data model
+* Support for parent/child relationships in the auth system
+* Flexible password hashing/bring your own algorithm
+* Better legacy backend support
+* Advanced registration forms
+* Unlimited SAML/OIDC connections
+* The ability to self-host
+* A truly free option with no limits on the number of users when you self host the community edition
+
```
Comment (dpolstra) [inline site/_posts/2021-03-10-migrate-from-auth0.md:37]: * Unlimited tenants

File: site/_posts/2021-03-10-migrate-from-auth0.md (hunk 1 above)
Comment (mooreds) [reply site/_posts/2021-03-10-migrate-from-auth0.md:3]: Migrating off of Auth0? Here's what you need to know

**Pull request merged**